frontend/server/src/Controllers/Submission.php

```diff
@@ -128,6 +128,56 @@ private static function createFeedback(
         }
     }
 
+    public static function tryCreateFeedback(
+        \OmegaUp\Request $r,
+        \OmegaUp\DAO\VO\Submissions $submission,
+        array $courseSubmissionInfo,
+        \OmegaUp\DAO\VO\Courses $course,
+        int|null $range_bytes_start,
+        int|null $range_bytes_end,
```

Comment:
La forma de representar que un parámetro puede cpntener `null` es de la siguiente forma:
```suggestion
        ?int $range_bytes_start,
        ?int $range_bytes_end,
```